src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -553,6 +564,14 @@ def _assess_jobs(self, all_jobs: list[BaseJob], all_clusters_by_id) -> list[JobI
                 job_details[job_key].success = 0
         return list(job_details.values())
 
+    def _safe_get_cluster_policy(self, policy_id: str) -> Policy | None:
+        try:
+            return self._ws.cluster_policies.get(policy_id)
+        except DatabricksError as err:
```

Comment:
Check for specific error code everywhere, not blindly ignoring all errors

Reply:
corrected